recipe/meta.yaml

```diff
@@ -146,7 +146,7 @@ outputs:
         - lsstdesc.coord
         - matplotlib-base !=3.6.0
         - mkl  # [linux64]
-        - moto >=3
+        - moto >=3,<5
```

Comment:
@timj upper pin was added here. Am I missing something here?

Reply:
Right. I should have scrolled further down on my phone.